deploy/materialized_view_activity.sql

```diff
@@ -4,16 +4,11 @@
 begin;
 
 create materialized view ggircs_swrs.activity as (
-  with x as (
-    select ghgr_import.id as ghgr_import_id,
-           ghgr_import.xml_file as source_xml
-    from ggircs_swrs.ghgr_import
-  )
-  select x.ghgr_import_id, activity_details.*
-  from x,
+  select id as ghgr_import_id, activity_details.*
```

Comment:
Creating an id column in the materialized view deploy file will conflict with the table exporter. The table exporter also creates an id column on each table when they are exported to the ggircs namespace.